composer.json

```diff
@@ -22,7 +22,6 @@
         "laminas/laminas-eventmanager": "^3.2",
         "laminas/laminas-servicemanager": "^3.3",
         "laminas/laminas-mvc": "^3.1",
-        "container-interop/container-interop": "^1.2",
         "php-amqplib/php-amqplib": "^3.1",
```

Comment:
Could you include the the `psr/container` requirement?

Reply:
Shall do. Thanks for the feedback.

Reply:
From reviewing the dependency chain, this is going to take a bit of time to complete. I'll work through it as quickly as is feasible.